poller-wrapper.py

```diff
@@ -86,7 +87,6 @@ def get_config_data():
     db_port = int(config['db_host'].rsplit(':')[1])
 else:
```

Comment:
I'm tempted to just remove this completely, as I don't see how it could work with the rest of librenms?